lib/elements/elements.css

```diff
@@ -194,3 +194,21 @@
   color: #eee;
   padding-left: 5px;
 }
+
+.toast {
+  padding: 0.5em 1em;
+  position: absolute;
+  right: 16px;
+  bottom: 0;
+  box-shadow: rgba(255, 255, 255, 0.1) 0 1px 0, rgba(0, 0, 0, 0.8) 0 1px 7px 0 inset;
+  min-width: 12em;
+  max-width: 40em;
+  background-color: inherit;
+
+  z-index: 10;
+  transition: 400ms;
+}
+
```

Comment:
maybe style this white or green to make it more visible ? I didn't notice the toast message, only after I knew it should exist and I was looking for it, I saw it.